src/data_handler/data_handler.cpp

```diff
@@ -18,14 +18,71 @@
 #include "wavelib.h"
 
 #include "FFTReal.h"
-
-
+#include "spdlog/sinks/null_sink.h"
+#include "spdlog/spdlog.h"
+#define LOGGER_NAME "brainflow_logger"
```

Comment:
lets rename it to smth like data_handler_logger or brainflow_data_handler_logger